### PR TITLE
{WIP} Faddats doc changes

### PR DESCRIPTION
**Pull request by abuehrle**
Edits to install page.

### CODE REVIEW

**Comment by abuehrle** [comment]
@fons that would be great.

**Comment by 2opremio** [comment]
@abuehrle I've done my best but I am not sure I've brought it to the state you wanted it to (there were duplicated commits, and a merge named `extra space` which was empty). I've created and pushed branch [`faddats-doc-changes-backup`](https://github.com/weaveworks/scope/tree/faddats-doc-changes-backup) with the original content in case you want to revert the changes or see its prior state.

**Comment by lukemarsden** [comment]
Changes LGTM. CI is sad but it looks transient, re-running.

File: site/installing.md [hunk 1]
```diff
@@ -63,6 +64,37 @@ This script downloads and runs a recent Scope docker image from the Docker Hub.
...
+     sudo chmod a+x /usr/local/bin/scope
+
+ **2. Then on the first node:**
+
+     scope launch 192.168.100.18 192.168.100.19 192.168.100.20
+
+ **3. And on the second node:**
+
+     scope launch 192.168.100.17 192.168.100.20 192.168.100.21
+     scope launch 192.168.100.17 192.168.100.18 192.168.100.21
```
Comment (lukemarsden) [inline site/installing.md]: There are 5 nodes in this example, shouldn't this say "And on the other nodes" and then give examples of how to construct the `scope launch` commands appropriately?

**Comment by abuehrle** [comment]
@lukemarsden I changed the wording slightly, but I'm still not sure if this is clear.  Do we peer each node to all of the other nodes. They way it's listed, it sounds like one takes a nodes and peers them all at once to all the others. Is that what's meant?

**Comment by 2opremio** [comment]
@lukemarsden Are you OK with the changes? This PR has been open for a month now.

**Comment by lukemarsden** [comment]
LGTM, sorry this took so long.

**Pull request merged**